log/sublogger.go

```diff
@@ -10,6 +10,8 @@ var errMultiWriterHolderIsNil = errors.New("multiwriter holder is nil")
 
 // NewSubLogger allows for a new sub logger to be registered.
 func NewSubLogger(name string) (*SubLogger, error) {
+
+	fmt.Println("REGISTERED NEW SUBLOGGER", name)
```

Comment:
Possibly leaked line?

Reply:
Yeah, there is a bunch of those. I will give it a good spit and polish and then request a review if you have some time.